src/.editorconfig

```diff
@@ -131,13 +132,15 @@ dotnet_diagnostic.S3247.severity=suggestion # Remove redundant cast - getting fa
 # Allow empty records for discriminated union types
 dotnet_diagnostic.S2094.severity=none # S2094: Classes should not be empty
 
-[**/tests/**/*.cs]
-# Allow our strange test class naming convention
-dotnet_naming_symbols.amt_tests.applicable_kinds = class,method
-dotnet_naming_symbols.amt_tests.word_separator = "_"
-dotnet_naming_symbols.amt_tests.capitalization = first_word_upper
-dotnet_diagnostic.IDE1006.severity=none
-dotnet_diagnostic.S101.severity=none
+# S2187: Test classes should contain at least one test case
+dotnet_diagnostic.S2187.severity = none
+
+[**/*Tests.cs]
+# S2187: Test classes should contain at least one test case
+dotnet_diagnostic.S2187.severity = none
```

Comment:
Oops, this message ended up here twice. I don't want to force restarting all of the tests just to address this one little thing. Maybe we can remove it in another ticket. Unless someone feels strongly, I'm going to leave it in place for now.